frontend/src/v5/ui/controls/fixedOrGrowContainer/index.ts

```diff
@@ -15,4 +15,4 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-export { FixedOrGrowContainer } from './fixedOrGrowContainer.component';
+export { FixedOrGrowContainer, FixedOrGrowContainerProps } from './fixedOrGrowContainer.component';
```

Comment:
This seems to cause WebPack to explode and I'm not too sure why, I think we can delete this index file anyway